app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Same here and as the last reviewer mentioned, please edit this line and create likes migration to make the foreign key `author_id` instead of `user_id`